[editor]/editor_main/server/mapEditorScriptingExtension_s.lua

```diff
@@ -62,11 +62,16 @@ end
 addEventHandler("onResourceStart", resourceRoot, onResourceStartOrStop)
 addEventHandler("onResourceStop", resourceRoot, onResourceStartOrStop)
 
-function receiveLODsClientRequest()
-	triggerClientEvent(client, "setLODsClient", resourceRoot, usedLODModels)
+function onPlayerResourceStart(resourceElement)

```

Comment:
Would a local function work?
```suggestion
local function onPlayerResourceStart(resourceElement)

```